### ISSUE TITLE
Error: release-please failed: Error updating ref heads/release-please--branches--main to

### ISSUE THREAD

**Issue by alyviam** [google-github-actions/release-please-action#712]
### TL;DR

After the first successful workflow run that integrated release-please into our repo, the workflow is failing to successfully create a release-please release PR after feature branches are merged to the main branch, failing with the error Error: release-please failed: Error updating ref heads/release-please--branches--main to <sha>

### Expected behavior

When feature branches are merged into main, release-please will successfully log the changes from the prior commit and create a release-please release PR with the new changes.

### Observed behavior

Release-please skips branch creation and then errors at the end of the workflow.

### Action YAML

```yaml
on:
  push:
    branches:
      - main
name: release-please
jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - name: "Create Release PRs"
        uses: google-github-actions/release-please-action@v3
        id: release
        with:
          token: <token>
          command: manifest
          release-type: node
          default-branch: main
          package-name: release-please-action
        # The logic below handles the npm publication:
      - name: "Publication"
        uses: actions/checkout@v3
        # These if statements ensure that a publication only occurs when
        # a new release is created:
        if: ${{ steps.release.outputs.releases_created }}
      - uses: actions/setup-node@v3
        with:
          node-version: 12
          registry-url: "https://npm.pkg.github.com/"
          scope: <scope>
        if: ${{ steps.release.outputs.releases_created }}
      - run: yarn install --frozen-lockfile
        env:
          token: <token>
        if: ${{ steps.release.outputs.releases_created }}
      - run: yarn workspaces run release
        env:
          token: <token>
        if: ${{ steps.release.outputs.releases_created }}
```

### Log output

```text
⚠ file packages/chat/npm-shrinkwrap.json did not exist
❯ Fetching packages/chat/samples/package.json from branch main
⚠ file packages/chat/samples/package.json did not exist
❯ Fetching packages/chat/CHANGELOG.md from branch main
❯ Fetching packages/chat/package.json from branch main
✔ updating from 1.1.0 to 1.2.0
✔ Starting GitHub PR workflow...
✔ Successfully found branch HEAD sha "<sha>".
✔ Skipping branch creation step...
✔ Got the latest commit tree
✔ Successfully created a tree with the desired changes with SHA <sha>
✔ Successfully created commit. See commit at <url>
✔ Updating reference heads/release-please--branches--main to <sha>
Error: release-please failed: Error updating ref heads/release-please--branches--main to <sha>
```
* Log output is too long

### Additional information

Some context into the issues we have had. 

We have recently integrated release-please as our publication workflow for our component library. We publish individual packages and so have created a `release-please-config` and `.release-please-manifest`. This is a mature repo as previously we used standard version for publication, because of that, we noted the `bootstrap-sha: <commit>`  in the `release-please-config` so that we didn't look too far back on the initial release-please release PR. However, since the initial merged release-please release PR we have been running into issues.

Context of prior issues, we removed the bootstrap sha key from the config file as the documentation states that it will be ignored after release-please has generated at least on release PR. However, this caused us to exceed the rate limit in the action. After adding the bootstrap-sha key with a defined commit back to the config file, we are now seeing release-please fail on the 'Create Release PRs' step with the error: Error: release-please failed: Error updating ref heads/release-please--branches--main to <sha>

**Comment by piotrw91** [comment]
Hey @alyviam what was the resolution for this problem? I'm facing the same error:  `Error updating ref heads/release-please--branches--main to <sha>`

**Comment by d-costa** [comment]
Sorry for commenting on ~1 year old issue, but I am also having this issue and I'd love to know how the author solved it.

**Comment by sillock1** [comment]
I'm also having this problem, did anybody find a solution to this?

**Comment by alyviam** [comment]
Hey everyone, our issue was the `bootstrap-sha` was added under the packages object. It is supposed to be top-level attribute.
